src/cloudai/systems/kubernetes/kubernetes_system.py

```diff
@@ -47,51 +56,61 @@ class KubernetesSystem(BaseModel, System):
     default_image: str
     scheduler: str = "kubernetes"
     global_env_vars: Dict[str, Any] = {}
+    monitor_interval: int = 1
     _core_v1: client.CoreV1Api
     _batch_v1: client.BatchV1Api
     _custom_objects_api: CustomObjectsApi
 
-    def __post_init__(self) -> None:
+    def __init__(self, **data):
```

Comment:
Since we are using Pydantic v2, let's use `model_post_init` method: https://stackoverflow.com/questions/66571079/alter-field-after-instantiation-in-pydantic-basemodel-class

Let's also add K8s specific cases that would check that these properties are initialized and not re-initialized on property access.

Reply:
> Since we are using Pydantic v2, let's use model_post_init method: https://stackoverflow.com/questions/66571079/alter-field-after-instantiation-in-pydantic-basemodel-class
Updated.

> Let's also add K8s specific test cases that would check that these properties are initialized and not re-initialized on property access.
I will do.

Reply:
> Let's also add K8s specific test cases that would check that these properties are initialized and not re-initialized on property access.

Updated, @amaslenn .